content/cover_letter.md

```diff
@@ -0,0 +1,26 @@
+Scott Markel
+Dassault Systèmes BIOVIA
+5005 Wateridge Vista Dr
+San Diego, CA 92121
+
+October Xth, 2020
+
+Dear Dr. Markel,
+
+We are writing to submit our manuscript entitled “Ten Quick Tip for Deep Learning in Biology” for consideration in _PLOS Computational Biology_'s education section.
+
+Ultimately, we have decided on the following ten tips as being critical for engaging in principled and effective DL-based research:
```

Comment:
we don't need to abbreviate DL here (no space constraints), and  I think this may make it a bit too informal.